submissions/krylenger/memory-pair-game/app.js

```diff
@@ -0,0 +1,240 @@
+const orelSong = new Audio('music/orel.mp3');
+const gopgopSong = new Audio('music/gopgop.mp3');
+const tanciSong = new Audio('music/tanci.mp3');
+const cadilacSong = new Audio('music/cadilac.mp3');
+const dimSong = new Audio('music/dim.mp3');
+
+const oneSecond = 1000;
+const totalPlayingCards = 10;
+const guessed = 'guessed';
+const greeting = `Congratulations! Let's play once more!`;
+
+let clickCounter = 0;
+let guessedCards = 0;
+let playedSongs = [];
+let playingCards = [];
+
+const cards = [
+    {
+        key: 1,
+        text: 'orel',
+        musicCard: false,
+        song: null
+    },
+    {
+        key: 1,
+        text: null,
+        musicCard: true,
+        song: orelSong
+    },
+    {
+        key: 2,
+        text: 'gopgop',
+        musicCard: false,
+        song: null
+    },
+    {
+        key: 2,
+        text: null,
+        musicCard: true,
+        song: gopgopSong
+    },
+    {
+        key: 3,
+        text: 'tanci',
+        musicCard: false,
+        song: null
+    },
+    {
+        key: 3,
+        text: null,
+        musicCard: true,
+        song: tanciSong
+    },
+    {
+        key: 4,
+        text: 'cadilac',
+        musicCard: false,
+        song: null
+    },
+    {
+        key: 4,
+        text: null,
+        musicCard: true,
+        song: cadilacSong
+    },
+    {
+        key: 5,
+        text: 'dim',
+        musicCard: false,
+        song: null
+    },
+    {
+        key: 5,
+        text: null,
+        musicCard: true,
+        song: dimSong
+    }
+]
+
+const main = document.querySelector('.main');
+let mainGridContainer = document.querySelector('.main__grid-container');
+const modal = document.querySelector('.modal');
+
+
+const styleNewCardFlipperFront = (newCardFlipperFront, card, cardNumber) => {
+    switch (cardNumber) {
```

Comment:
I see a lot of similar code here. It seems we can switch only class name which depends on the card number.
So let's simplify it by creating some variable at the beginning.
For example:
``` javascript
let generatedClassName;
switch(cardNumber) {
case 0:
case 9:
generatedClassName = 'card--circle-img'
case 1:
...
case 8:
generatedClassName = 'card--vertical-line-img'
...
newCardFlipperFront.classList.add('card__flipper-front', card.key, generatedClassName);
```

Reply:
wow, I get it! It looks much cleaner now, thanks!
✅ Done